src/SFA.DAS.PSRService.Domain/Entities/Report.cs

```diff
@@ -97,6 +121,59 @@ private ReportingPercentages GetPercentages()
             if (apprenticePeriod != 0 & employmentStart != 0)
                 percentages.NewThisPeriod = ((apprenticePeriod / employmentStart) * 100).ToString("F2");
 
+            if (IsLocalAuthority.HasValue)
+                TotalEmployees = (employmentPeriod + employmentEnd).ToString("F0");
+
+            if (ReportingPercentages != null)
+                percentages.Title = ReportingPercentages.Title;
+
+            return percentages;
+
+        }
+        private ReportingPercentages GetPercentagesSchools()
+        {
+            var percentages = new ReportingPercentages();
+
+            if (Sections == null)
+                return null;
+
+            var employeeQuestions = GetQuestionSection("SchoolsEmployees");
+
+            if (employeeQuestions == null)
+                return null;
+
+            var apprenticeQuestions = GetQuestionSection("SchoolsApprentices");
+
+            if (apprenticeQuestions == null)
+                return null;
+
+
+            decimal employmentPeriod = 0, apprenticePeriod = 0, employmentEnd = 0, apprenticeEnd = 0, employmentStart = 0;
+
+            if (String.IsNullOrWhiteSpace(employeeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer) == false)
+                employmentPeriod = decimal.Parse(employeeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer);
+            if (String.IsNullOrWhiteSpace(apprenticeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer) == false)
+                apprenticePeriod = decimal.Parse(apprenticeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer);
+
+            if (String.IsNullOrWhiteSpace(employeeQuestions.Questions.Single(w => w.Id == "atEnd").Answer) == false)
+                employmentEnd = decimal.Parse(employeeQuestions.Questions.Single(w => w.Id == "atEnd").Answer);
+            if (String.IsNullOrWhiteSpace(apprenticeQuestions.Questions.Single(w => w.Id == "atEnd").Answer) == false)
+                apprenticeEnd = decimal.Parse(apprenticeQuestions.Questions.Single(w => w.Id == "atEnd").Answer);
+
+            if (String.IsNullOrWhiteSpace(employeeQuestions.Questions.Single(w => w.Id == "atStart").Answer) == false)
+                employmentStart = decimal.Parse(employeeQuestions.Questions.Single(w => w.Id == "atStart").Answer);
+
+
+            if (apprenticePeriod != 0 & employmentPeriod != 0)
+                percentages.EmploymentStarts = ((apprenticePeriod / employmentPeriod) * 100).ToString("F2");
+
+            if (apprenticeEnd != 0 & employmentEnd != 0)
+                percentages.TotalHeadCount = ((apprenticeEnd / employmentEnd) * 100).ToString("F2");
+
+            if (apprenticePeriod != 0 & employmentStart != 0)
+                percentages.NewThisPeriod = ((apprenticePeriod / employmentStart) * 100).ToString("F2");
+
+            percentages.Title = ReportingPercentagesSchools.Title;
```

Comment:
as almost all of this code is copy and pasted from the above method, is it worth re-using?